lib/segment/src/index/field_index/full_text_index/tokenizers/mod.rs

```diff
@@ -170,13 +174,11 @@ fn truncate_cow_ref<'a>(inp: &Cow<'a, str>, len: usize) -> Cow<'a, str> {
 #[derive(Debug, Clone)]
 pub struct Tokenizer {
     tokenizer_type: TokenizerType,
-    min_token_len: Option<usize>,
-    max_token_len: Option<usize>,
     config: TokenizerConfig,
 }
 
 impl Tokenizer {
-    pub fn new(params: &TextIndexParams) -> Self {
+    pub fn new_from_text_index_params(params: &TextIndexParams) -> Self {
```

Comment:
Might be worth considering this a as a `From<&TextIndexParams> for Tokenizer` implementation